litgpt/api.py

```diff
@@ -221,6 +223,11 @@ def generate(
         prompt_length = input_ids.size(0)
         max_returned_tokens = prompt_length + max_new_tokens
 
+        first_turn = self.model.mask_cache is None
```

Comment:
Now you need to update `test_generate` from `test_api` with three prompts (instead of one):
1. the current one
2. a longer than the current one
3. a shorter that the previous one

Reply:
Added. Does it look ok to you know @Andrei-Aksionov ?

Reply:
Yes, thank you.